awx_collection/plugins/modules/export.py

```diff
@@ -28,52 +28,52 @@
       default: 'False'
     organizations:
       description:
-        - organization name to export
-      type: str
+        - organization names to export
+      type: list
```

Comment:
@john-westcott-iv do we have a convention for when a value could be a str or a list?

Reply:
From some testing it seems like ansible is ok with still getting a string here even though we specified it as a list. I was kind of hoping it would be like "I'll make a list with one string element" but that does not seem to be the case. So in the module code we may want to do a check and make it a list of one item if its just a string.

Also, we can add: `elements: str` to all of these to ensure that we are getting strings in the list (see other modules as examples).

Reply:
I did two tests like that:

1. List
```yaml
     name: Export asserts
      awx.awx.tower_export:
        tower_host: url
        tower_oauthtoken: token
        projects: ['abc1_project_netcool', 'AIC_Project_Rockesh']
        validate_certs: no
      register: data
```
3. String
```yaml    
    name: Export asserts
      awx.awx.tower_export:
        tower_host: url
        tower_oauthtoken: token
        projects: abc1_project_netcool
        validate_certs: no
      register: data
 ```
In both cases projects are exported fine. So the list type can handle strings and lists without any issues it seems. Do we really need to create a list of one string? Later a sting or list will be handled correctly in `_filtered_list`

Reply:
Looks like we just need to add the `elements: str` to the docs section for all of the list types to coincide with the argument spec and then we should be good to go.

Reply:
@jbradberry thanks for adding this small change instead of me 😃